Add new attributes about permanent project owner

ClaraRosalie opened the pull request:
Fixes #97

In #65 we proposed to add new questions about the name and email address of the permanent project owner/data owner. The two new questions were already implemented in [rdmorganiser/questions/rdmo.xml](https://github.com/FDM-UARuhr/rdmo-catalog-uaruhr/blob/add-new-attributes/rdmorganiser/questions/rdmo.xml) via #75. This PR adds the related attributes `project/dataset/preservation/responsible_person/name` and `project/dataset/preservation/responsible_person/email` to [rdmorganiser/domain/rdmo.xml](https://github.com/FDM-UARuhr/rdmo-catalog-uaruhr/blob/add-new-attributes/rdmorganiser/domain/rdmo.xml).

ClaraRosalie committed:
add attributes

jmfrenzel commented on rdmorganiser/domain/rdmo.xml:
```diff
@@ -1498,4 +1498,18 @@
+	  <key>responsible_person_name</key>
+	  <path>project/dataset/preservation/responsible_person/name</path>
+	  <dc:comment>Refers to the permanent project owner see #65</dc:comment>
+	  <parent dc:uri="https://rdmorganiser.github.io/terms/domain/project/dataset/sharing"/>
```
```suggestion
	  <parent dc:uri="https://rdmorganiser.github.io/terms/domain/project/dataset/preservation/responsible_person"/>
```

jmfrenzel commented on rdmorganiser/domain/rdmo.xml:
```diff
@@ -1498,4 +1498,18 @@
+	  <key>responsible_person_email</key>
+	  <path>project/dataset/preservation/responsible_person/email</path>
+	  <dc:comment>Refers to the permanent project owner see #65</dc:comment>
+	  <parent dc:uri="https://rdmorganiser.github.io/terms/domain/project/dataset/sharing"/>
```
```suggestion
	  <parent dc:uri="https://rdmorganiser.github.io/terms/domain/project/dataset/preservation/responsible_person"/>
```

jmfrenzel reviewed (changes requested):
As of https://github.com/FDM-UARuhr/rdmo-catalog-uaruhr/issues/65#issuecomment-842897515 we would attach the two new questions the parent attribute `"https://rdmorganiser.github.io/terms/domain/project/dataset/preservation/responsible_person"`
which not existing and we need to create it, too.   
The new `"https://rdmorganiser.github.io/terms/domain/project/dataset/preservation/responsible_person"` would reside under 
`https://rdmorganiser.github.io/terms/domain/project/dataset/preservation` .

jmfrenzel commented on rdmorganiser/domain/rdmo.xml:
```diff
@@ -1498,4 +1498,18 @@
+	  <dc:comment>Refers to the permanent project owner see #65</dc:comment>
+	  <parent dc:uri="https://rdmorganiser.github.io/terms/domain/project/dataset/sharing"/>
+	</attribute>
 </rdmo>
```
```suggestion
	<attribute dc:uri="https://rdmorganiser.github.io/terms/domain/project/dataset/preservation/responsible_person">
	  <uri_prefix>https://rdmorganiser.github.io/terms</uri_prefix>
	  <key>responsible_person</key>
	  <path>project/dataset/preservation/responsible_person</path>
	  <dc:comment>Refers to the permanent project owner see #65</dc:comment>
	  <parent dc:uri="https://rdmorganiser.github.io/terms/domain/project/dataset/preservation"/>
	</attribute>
</rdmo>
```

jmfrenzel committed:
Apply suggestions from code review

Add Fix parent attribute domain/project/dataset/preservation/responsible_person

jmfrenzel reviewed (commented):
Ok, reviewed paring with @ClaraRosalie
